client-common/src/tendermint/client.rs

```diff
@@ -25,9 +25,16 @@ pub trait Client: Send + Sync {
         heights: T,
     ) -> Result<Vec<BlockResults>>;
 
+    /// Fetch continuous blocks and verify them.
+    fn block_batch_verified<'a, T: Clone + Iterator<Item = &'a u64>>(
+        &self,
+        _state: lite::TrustedState,
```

Comment:
```suggestion
        state: lite::TrustedState,
```

Reply:
Adding a underscore is to remove the unused variable warning.

Reply:
This is a trait. I don't think you'll get an unused variable warning in traits.